### DISCUSSION TITLE
[Unstable] Multi-tool issues

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14624] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

* still stackable to 8 in inventory, 32 in cabinets
* has quality property but no required skill for crafting
* not swingable as a melee weapon
* holdpos is weird

![image](https://github.com/user-attachments/assets/4dd601d5-d47f-456b-ac42-9590356f7a86)
![image](https://github.com/user-attachments/assets/08483c8d-263b-4ff1-a958-0b1c1bfa0419)

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by JeroonBosch** [comment]
Thanks for your report!

> not swingable as a melee weapon

We'll keep this as an intended weakness, but the rest will be fixed.

**Comment by JeroonBosch** [comment]
Closing as the fixes have been merged internally (not yet on unstable)
